src/lib/viewers/image/ImageViewer.js

```diff
@@ -44,6 +63,10 @@ class ImageViewer extends ImageBaseViewer {
         this.imageEl.classList.add(CLASS_INVISIBLE);
 
         this.currentRotationAngle = 0;
+
+        if (this.options.enableAnnotationsImageDiscoverability) {
+            this.addListener('zoom', this.handleZoomEvent);
```

Comment:
Is there an existing constant for this?

Reply:
I couldn't find an existing constant for 'zoom'.